### PR TITLE
fix: support blank space in field_value and escape character in Identifier

### LINKED ISSUE #243: ts-cli write field_value with blank space failed
### Describe the bug（Bug 描述）

ts-cli write field_value with blank space failed.

image

second:

image

### To Reproduce（Bug 复现步骤）

1. start ts-server.
2. ./build/ts-cli
3. create database db0
4. use db0
5. insert weather,location=us-midwest temperature="too warm"
    
   **ERR: error parsing query: syntax error: unexpected IDENT**

6. insert weather,location=us-midwest temperature="toowarm"
7. select * from weather
```
name: weather
+---------------------+------------+-------------+
| time                | location   | temperature |
+---------------------+------------+-------------+
| 1682393021308851000 | us-midwest | toowarm     |
+---------------------+------------+-------------+
3 columns, 1 rows in set
```

### Expected behavior（期望结果）

`insert weather,location=us-midwest temperature="too warm"`

can write to ts-server successfully.

### Screenshots（屏幕截图）

_No response_

### Logs（完整的错误日志）

_No response_

### Additional context（其他的一些补充内容）

_No response_

### PR DESCRIPTION
**Pull request by Benevor**
### What problem does this PR solve?

Issue Number: fix #243

### What is changed and how it works?
Modify the ts-cli parser and write corresponding test cases to address the bugs mentioned in the corresponding issue and any additional bugs discovered during writing.

### How Has This Been Tested?

Add corresponding test cases.
Rebuild the project, manually enter SQL statements, and execute successfully.

- [ ] Test A
- [ ] Test B
- [x] Test cases to be added
- [ ] No code

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/ts-cli/geminiql/parser.y [hunk 1]
```diff
@@ -211,12 +211,32 @@ LINE_PROTOCOL:
...
+        $$ = $1 + " " + $2
+    }
+
 TIME_SERIE:
     MEASUREMENT COMMA KV_RAWS KV_RAWS
     {
         $$ = $1 + $2 + $3 + " " + $4
     }
 
+TIME_SERIE_WITHOUT_INTO:
```
Comment (shilinlee) [inline app/ts-cli/geminiql/parser.y:230]: It may not be necessary to increase the `*_without_into` type. Add `|MEASUREMENT KV_RAWS` to `TIME_SERIE`.

**Review by shilinlee — CHANGES_REQUESTED** [review]

File: app/ts-cli/geminiql/parser.y (hunk 1 above)
Comment (Benevor) [reply app/ts-cli/geminiql/parser.y]: Yes, it's really not necessary.I have modified the code.

File: app/ts-cli/geminiql/lexer.go [hunk 2]
```diff
@@ -133,6 +133,14 @@ func (t *Tokenizer) scanIdentifier() (int, string) {
 		ch := t.read()
 		if ch == EOF {
 			break
+		} else if ch >= 128 {
```
Comment (shilinlee) [inline app/ts-cli/geminiql/lexer.go:136]: This way of writing supports `emoji` is inappropriate.

**Review by shilinlee — CHANGES_REQUESTED** [review]

File: app/ts-cli/geminiql/lexer.go (hunk 2 above)
Comment (Benevor) [reply app/ts-cli/geminiql/lexer.go]: I have completed the modifications according to the above plan.

**Review by shilinlee — APPROVED** [review]

**Pull request merged**